.github/ISSUE_TEMPLATE/release.md

```diff
@@ -63,15 +63,22 @@ Note: Usually update client first then runtime.
 
 Note: Do not publish draft releases from PR branches, because those branches will be deleted when the PR is merged.
 
-### After Runtime Upgrade
-- [ ] Notify subscan team. Ensure subscan service can continue to scan calamari blocks.
+### Before Runtime Upgrade Vote
+- [ ] Prepare a governance post and submit to our forum with description and motivation for changes.
+
+### During Runtime Upgrade Vote
+- Notify all external users of `manta`, include the block number the upgrade is enacted!
+  - [ ] [Calamari Network Forum](https://forum.manta.network/c/calamari-network-governance/6)
+  - [ ] Manta/Calamari Discord Announcement
+  - [ ] Exchange Integration Teams
```

Comment:
@Dengjianping Please add the concrete Exchanges and their contact method here ( if public )

Reply:
I have added a notion page.
https://www.notion.so/mantanetwork/Exchanges-3rd-Infrastructures-b089e136a14b430ea405400311b362cb